airbyte-integrations/connectors/source-file/source_file/spec.json

```
@@ -10,26 +10,31 @@
     "properties": {
       "dataset_name": {
         "type": "string",
-        "description": "Name of the final table where to replicate this file (should include only letters, numbers dash and underscores)"
+        "title": "Dataset Name",
+        "description": "The Name of the final table to replicate this file in (should include letters, numbers dash and underscores only)."
       },
       "format": {
         "type": "string",
         "enum": ["csv", "json", "jsonl", "excel", "feather", "parquet"],
         "default": "csv",
-        "description": "File Format of the file to be replicated (Warning: some format may be experimental, please refer to docs)."
+        "title": "Format",
+        "description": "The Format of the file which should be replicated (Warning: some formats may be experimental, please refer to the docs)."
       },
       "reader_options": {
         "type": "string",
-        "description": "This should be a valid JSON string used by each reader/parser to provide additional options and tune its behavior",
+        "title": "Reader Options",
+        "description": "This should be a string in JSON format and depended on the chosen file format to provide additional options and tune its behavior.",
```

Can you add a link to help the user understand what are the possible values/options they can configure here? 
```suggestion
        "description": "This should be a string in JSON format. It depends on the chosen file format to provide additional options and tune its behavior.",
```